.github/workflows/pr_metadata.yml

```diff
@@ -0,0 +1,27 @@
+# Auto-update metadata on pull requests
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: "Pull Request Metadata"
+on:
+- pull_request_target
+
+permissions:
+  pull-requests: write
+
+jobs:
+  Triage:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Assign Author
+      # Add the author as an assignee of the PR
+      # https://github.com/marketplace/actions/auto-author-assign
+      uses: toshimaru/auto-author-assign@v1.3.5
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Maybe better to use a specific commit hash instead of a version so that the `v1.3.5` tag isn't unexpectedly swapped out with a malicious commit?

Reply:
Done in 80caa45